vllm/model_executor/layers/quantization/utils/w8a8_utils.py

```diff
@@ -113,7 +113,9 @@ def apply_fp8_linear(
     #   If static, layer.input_scale is scalar and x_scale is input_scale.
 
     if cutlass_fp8_supported:
-        qinput, x_scale = ops.scaled_fp8_quant(input, input_scale)
+        qinput, x_scale = ops.scaled_fp8_quant(input,
+                                               input_scale,
+                                               use_per_token_if_dynamic=True)
```

Comment:
Shouldn't `use_per_token_if_dynamic` be set by a scheme or something? I don't see why it should always be true for this case

Reply:
In what case would we not want to use dynamic per token?

Reply:
@mgoin the hypothesis here is that dynamic per token is an overall win over dynamic per tensor when supported. Higher accuracy, but also easier to fuse RMSNorm + Quant, and fewer dependencies so better parallelization for the quantize kernels overall. Downside is more scales.

We'll need to benchmark, but IMO ok for this PR

Reply:
I wouldn't assume a decent performance hit -- the overheads from the CUTLASS epilogues are quite small (like 3%), and there are advantages to doing per-token when quantizing as well. We'll measure.

Reply:
I changed my mind -- as I am implementing the per-token/per-channel wrapper for `torch._scaled_mm`, I think it would be nicer to grab this from a config somewhere

Reply:
I tried to pass this from the scheme. However, this became very hard because I needed to check if cutlass is supported in multiple places

I think that deciding this here is the right place